hypervisor/Makefile

```diff
@@ -366,6 +373,31 @@ install-debug: $(HV_OBJDIR)/$(HV_FILE).map $(HV_OBJDIR)/$(HV_FILE).out
 	install -D $(HV_OBJDIR)/$(HV_FILE).out $(DESTDIR)/usr/lib/acrn/$(HV_FILE).$(BOARD).$(FIRMWARE).$(SCENARIO_NAME).out
 	install -D $(HV_OBJDIR)/$(HV_FILE).map $(DESTDIR)/usr/lib/acrn/$(HV_FILE).$(BOARD).$(FIRMWARE).$(SCENARIO_NAME).map
 
+cfg_src: $(VERSION)
+ifneq ($(BOARD_FILE)$(SCENARIO_FILE),)
+	@if [ ! -f $(BOARD_FILE) ]; then \
+		echo "Board xml file $(BOARD_FILE) is not exist!"; exit 1; \
+	fi
+	@if [ ! -f "$(SCENARIO_FILE)" ]; then \
+		echo "Scenario xml file $(SCENARIO_FILE) is not exist!"; exit 1; \
+	fi
+	@if [ "$(BOARD)" != "$(CONFIG_BOARD)" ]; then \
+		echo "Board in xml file <$(BOARD)> does not match Kconfig <$(CONFIG_BOARD)>!"; exit 1; \
+	fi
+	@if [ "$(SCENARIO_IN_XML)" != "$(SCENARIO_NAME)" ]; then \
+		echo "Scenario in xml file <$(SCENARIO_IN_XML)> does not match Kconfig <$(SCENARIO_NAME)>!"; exit 1; \
+	fi
+	@python3 ../misc/acrn-config/board_config/board_cfg_gen.py --board $(BOARD_FILE) --scenario $(SCENARIO_FILE) > $(HV_OBJDIR)/.cfg_src_result
+	@cat $(HV_OBJDIR)/.cfg_src_result
+	@if [ "`sed -n /successfully/p $(HV_OBJDIR)/.cfg_src_result`" = "" ]; then exit 1;	fi
+	@python3 ../misc/acrn-config/scenario_config/scenario_cfg_gen.py --board $(BOARD_FILE) --scenario $(SCENARIO_FILE) > $(HV_OBJDIR)/.cfg_src_result
+	@cat $(HV_OBJDIR)/.cfg_src_result
+	@if [ "`sed -n /successfully/p $(HV_OBJDIR)/.cfg_src_result`" = "" ]; then exit 1;	fi
+	@echo "Import hypervisor configurations from Config-xmls, configurations in source code are ignored!"
+else
+	@echo "Use hypervisor configurations from source code directly."
```

Comment:
Does the hypervisor configurations from the source code equals to arch/x86/configs/$(CONFIG_BOARD)/platform_acpi_info.h ?

Reply:
The platform_acpi_info.h is just one of configurations.
In current code base, platform_acpi_info.h might not be exist for some $(CONFIG_BOARD). In this case, we need to copy the template from arch/x86/configs/platform_acpi_info.h as its acpi info file.

Reply:
OK, please address the other comment and update the PR, then I can approve it. Thanks.